src/server/src/main/docker/Dockerfile

```diff
@@ -11,7 +11,7 @@ ENV REAPER_SEGMENT_COUNT=200 \
     REAPER_STORAGE_TYPE=memory \
     REAPER_ENABLE_CORS=true \
     REAPER_INCREMENTAL_REPAIR=false \
-    REAPER_ALLOW_UNREACHABLE_NODES=false \
+    REAPER_ENABLE_DYNAMIC_SEEDS=ALL \
```

Comment:
`REAPER_ENABLE_DYNAMIC_SEEDS=ALL` should be `REAPER_DATACENTER_AVAILABILITY=ALL`

Reply:
fixed! thanks for the spot!